delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/indexing/BlazegraphIndexingStreamEntry.scala

```diff
@@ -31,15 +34,15 @@ final case class BlazegraphIndexingStreamEntry(
     * Deletes the current resource named graph (and all its triples)
     */
   def delete(): Task[SparqlWriteQuery] =
-    namedGraph(resource.resolvedId).map(SparqlWriteQuery.drop)
+    namedGraph(resource.id).map(SparqlWriteQuery.drop)
 
   /**
     * Generates an Sparql replace query with all the triples to be added to the resource named graph
     */
   def index(includeMetadata: Boolean): Task[Option[SparqlWriteQuery]] =
     for {
-      triples    <- toTriples(includeMetadata)
-      namedGraph <- namedGraph(resource.resolvedId)
+      triples    <- IO.fromEither(toTriples(includeMetadata))
+      namedGraph <- namedGraph(resource.id)
```

Comment:
Why this is reverted? To use `resource.resolvedId` was a fix for projects.

Reply:
When you create `IndexingData` from `ResourceF` it uses `resolvedId`.